api/HTMLAnchorElement.json

```diff
@@ -268,10 +268,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": false
```

Comment:
So I realize this patch is scoped to just updating the Safari data — but as far as I can see, all the data we have for this feature is wrong. There’s no `HTMLAnchorElement.prototype.media` in Blink or Gecko either — which is unsurprising since https://html.spec.whatwg.org/multipage/text-level-semantics.html#htmlanchorelement doesn’t actually define a `media` member for `HTMLAnchorElement`, not even in as an obsolete member in https://html.spec.whatwg.org/multipage/obsolete.html#HTMLAnchorElement-partial. And there is actually no https://developer.mozilla.org/docs/Web/API/HTMLAnchorElement/media article.

So it seems to me we should just remove this entire feature.

Reply:
Agree that this should be removed. I think it's worth applying this change first, however, since (eventually) the feature would then show up in a listing of features not supported anywhere. Per our data it's still in Edge and Firefox, and we should check that before removal.

@vinyldarkscratch can you split this out into a separate PR?

Reply:
Just to confirm: by splitting out, are you talking about this change, or removing the feature?

Reply:
I mean doing the removal separately. Setting this to false here is fine.

Reply:
Gotcha, thanks for the clarification!  Definitely, I'll get a PR for that.